### PR TITLE
Add requirement to fetch behave source code and update readme

### PR DESCRIPTION
**Pull request by algochoi**
This PR makes a README change to add details to the development section and specify the `requirements.txt` to fetch the latest `behave` for tag support.

Thanks to @barnjamin for this suggestion 😃

### CODE REVIEW

**Review by barnjamin — APPROVED** [review]
Thanks!

File: README.md [hunk 1]
```diff
@@ -14,7 +14,16 @@ Alternatively, choose a [distribution file](https://pypi.org/project/py-algorand
...
 ## SDK Development
 
-Run tests with `make docker-test`
+Install dependencies
+* `pip install -r requirements.txt`
+
+Run tests
+* `make docker-test`
+* `make unit` (only unit tests)
+* `make integration` (only integration tests)
```
Comment (jasonpaulos) [inline README.md:23]: This is a little misleading because these commands only work if the testing harness is running.

**Review by jasonpaulos — COMMENTED** [review]
Since it's probably best to specify dependencies only in one place, could also you modify line 10 of `Dockerfile` to use `pip install -r requirements.txt -q` instead?

File: README.md (hunk 1 above)
Comment (algochoi) [reply README.md:23]: Good point -- I think I'll just revert this back to `make docker-test`

**Review by jasonpaulos — APPROVED** [review]

**Pull request merged**